lib/filesystem/Filesystem.cpp

```diff
@@ -126,10 +126,10 @@ ISimpleResourceLoader * CResourceHandler::createInitial()
 {
 	//temporary filesystem that will be used to initialize main one.
 	//used to solve several case-sensivity issues like Mp3 vs MP3
-	auto initialLoader = new CFilesystemList();
+	auto *initialLoader = new CFilesystemList();
```

Comment:
Not the only occurence
```suggestion
	auto * initialLoader = new CFilesystemList();
```

Reply:
done